pages/common/gh-api.md

```diff
@@ -0,0 +1,32 @@
+# gh api
+
+> Makes authenticated HTTP requests to the GitHub API and prints the response.
+> More information: <https://cli.github.com/manual/gh_api>.
+
+- Display the releases for the repository in the local path in JSON format:
+
+`gh api repos/:owner/:repo/releases`
+
+- Create a reaction for a specific issue:
+
+`gh api --header {{Accept:application/vnd.github.squirrel-girl-preview+json}} repos/:owner/:repo/issues/{{issue_number}}/reactions --raw-field {{content=+1}}`
```

Comment:
How about this?
```suggestion
`gh api --header {{Accept:application/vnd.github.squirrel-girl-preview+json}} {{repos/:owner/:repo/issues/:issue_number/reactions}} --raw-field {{content=+1}}`
```

Reply:
This also confuses me a bit&mdash;how do you specify which comment to react to?

Reply:
To create a reaction for an issue comment you should use [another endpoint](https://docs.github.com/en/free-pro-team@latest/rest/reference/reactions#create-reaction-for-an-issue-comment): `/repos/:owner/:repo/issues/comments/{{comment_id}}/reactions`

Reply:
Placeholder values like `:owner` and `:repo` in the endpoint get replaced with values from the repository of the current directory. That's why we can't use `:issue_number` because we're in conflict with the `gh api` syntax.
```suggestion
`gh api --header {{Accept:application/vnd.github.squirrel-girl-preview+json}} --raw-field '{{content=+1}}' {{repos/:owner/:repo/issues/123/reactions}}`
```

Reply:
Ah okay, I see.